next.config.js

```diff
@@ -1,4 +1,15 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+    images: {
+        remotePatterns: [
+          {
+            protocol: 'https',
+            hostname: 'lh3.googleusercontent.com',
+            port: '',
+            // pathname: '/account123/**',
```

Comment:
If the commented out code is not needed maybe get rid of it?

Reply:
Noted